perses/app/relative_point_mutation_setup.py

```diff
@@ -159,23 +159,31 @@ def __init__(self,
 
         # Load the ligand, if present
         molecules = []
-        if ligand_file:
-            if ligand_file.endswith('.sdf'): # small molecule
-                ligand_mol = createOEMolFromSDF(ligand_file, index=ligand_index)
-                ligand_mol = generate_unique_atom_names(ligand_mol)
+        if ligand_input:
+            if isinstance(ligand_input, str):
+                if ligand_input.endswith('.sdf'): # small molecule
+                        ligand_mol = createOEMolFromSDF(ligand_input, index=ligand_index)
+                        ligand_mol = generate_unique_atom_names(ligand_mol)
```

Comment:
This line is redundant, as `generate_unique_atom_names is called here: https://github.com/choderalab/perses/blob/f640bf82d75262530e68255f4d4d5fc43f70eea3/perses/utils/openeye.py#L354

Reply:
@dominicrufa can we remove it?

Reply:
yep. remove in the `relative_point_mutation_setup.py`, you mean?

Reply:
yes in `relative_point_mutation_setup.py`